### PR TITLE
+ Lwt4

### PR DESCRIPTION
**Pull request by pirbo**
Depends upon #98

### CODE REVIEW

File: packages/lwt-windows.4.2.1/opam [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  "result-windows"
+  "seq-windows"
+
+  "bisect_ppx" {dev & >= "1.3.0"}
+  "ocamlfind" {dev & >= "1.7.3-1"}
+]
+depopts: [
+  "base-threads"
+  "base-unix"
+  "conf-libev"
```
Comment (toots) [inline packages/lwt-windows.4.2.1/opam]: Does this work?

**Comment by toots** [comment]
From a local run:
```
% ./tests/run_tests.sh
Building ocamlcross/windows-x64-pretest:4.07.0..[ok]✅

From github.com:ocaml-cross/opam-cross-windows
 * branch            master     -> FETCH_HEAD

Building lwt-windows.4.2.1 using ocamlcross/windows-x64-pretest:4.07.0.. [ok]✅
Building mmap-windows.1.1.0 using ocamlcross/windows-x64-pretest:4.07.0.. [ok]✅
```

**Comment by pirbo** [comment]
Sorry, keeping the `depopts` was a mistake. I removed them and force pushed to keep it in 1 commit...

**Comment by toots** [comment]
Looks great, thanks!

**Pull request merged**